sdk/core/core-paging/src/getPagedAsyncIterator.ts

```diff
@@ -42,19 +42,8 @@ async function* getItemAsyncIterator<TElement, TPage, TLink, TPageSettings>(
   pagedResult: PagedResult<TPage, TPageSettings, TLink>
 ): AsyncIterableIterator<TElement> {
   const pages = getPageAsyncIterator(pagedResult);
-  const firstVal = await pages.next();
```

Comment:
I like that we are removing this! Is Text Analytics the only library currently using this functionality? If so I'm not too nervous about the potential behavior change.

Are you planning a major or minor version bump?

Reply:
I am thinking of keeping it a minor version bump since I am the one who added this logic before and I don't think anyone else relies on it. Is there any rest-client that you would like to test this change against?

Reply:
Doesn't this have to be a major bump since otherwise older text analytics clients would pull it in?

Reply:
silly me 😅, you're right. In this case, we could shelf this change for now and lump it with anything else we would like to do in a future v2. And of course, unless anyone feels strongly about doing it now :)

Reply:
Yeah if there is a "v2 wishlist" this would be a good item to have

Reply:
I opened https://github.com/Azure/azure-sdk-for-js/issues/21836